client/config.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Not mandatory but is LB boilerplate. I opted to stay as close to LB generated structure as possible.